app/res/values/dimens.xml

```diff
@@ -26,6 +26,12 @@
     <dimen name="min_button_height">52dp</dimen>
     <dimen name="spacer">10dp</dimen>
 
+    <dimen name="menu_icon_size">60dp</dimen>
+    <dimen name="menu_grid_icon_size">120dp</dimen>
+    <dimen name="menu_grid_icon_size_with_badge">136dp</dimen>
+    <dimen name="numeric_badge_font_size">15dp</dimen>
+    <dimen name="numeric_badge_width">30dp</dimen>

```

Comment:
@ctsims I was previously using `sp` for `numeric_badge_font_size` and `numeric_badge_width` because generally we want `sp` for text (and the badge_width has to be based on the font size), but based on the conversation we just had, I think we may actually want to use `dp` here instead, so that the size of the badge stays proportional with the size of the entire menu item on different screens. Does that sound right to you?